compiler/src/main/java/com/squareup/anvil/compiler/RealAnvilContext.kt

```diff
@@ -0,0 +1,8 @@
+package com.squareup.anvil.compiler
```

Comment:
That file belongs into the `codegen` package.

Reply:
Even though this is only used in the AnvilComponentRegistrar?

Reply:
Yes, the interface is in the same package and it makes sense to me.

Reply:
Done!